crates/common_enums/src/enums.rs

```diff
@@ -1754,16 +1754,26 @@ pub enum MandateStatus {
 )]
 #[router_derive::diesel_enum(storage_type = "text")]
 pub enum CardNetwork {
+    #[serde(alias = "VISA")]
     Visa,
+    #[serde(alias = "MASTERCARD")]
     Mastercard,
+    #[serde(alias = "AMERICANEXPRESS")]
     AmericanExpress,
     JCB,
+    #[serde(alias = "DINERSCLUB")]
     DinersClub,
+    #[serde(alias = "DISCOVER")]
     Discover,
+    #[serde(alias = "CARTRSBANCAIRES")]
     CartesBancaires,
```

Comment:
Was this supposed to be `CARTESBANCAIRES`?

```suggestion
    #[serde(alias = "CARTESBANCAIRES")]
    CartesBancaires,
```